tic-tac-toe-the3rd/src/components/Cell.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+function Cells(props) {
```

Comment:
One small note - you can use de-structuring here if you like so you don't need to write `props.` in the markup.

Reply:
```suggestion
function Cells({click, value}) {
```

Reply:
One other tiny thing - I'd name this component `Cell` instead of `Cells` since it really just represents one cell. I'd expect a `Cells` component to render a whole list of cells.